e2e/angular-core/src/projects.test.ts

```diff
@@ -111,6 +111,14 @@ describe('Angular Projects', () => {
 
     // port and process cleanup
     await killProcessAndPorts(process.pid, appPort);
+
+    const esbProcess = await runCommandUntil(
+      `serve ${esbuildApp} -- --port=${appPort}`,
+      (output) => output.includes(`listening on localhost:4207`)
```

Comment:
```suggestion
      (output) => output.includes(`listening on localhost:${appPort}`)
```

Reply:
ah i think there's another line in the file that could benefit from this change, i had just copied that line. this is better